### ISSUE TITLE
Support 2 Offers Properties:

### ISSUE THREAD

**Issue by ollie10** [RehanSaeed/Schema.NET#589]
### Describe the feature

Hello, I was wondering if you plan to support two properties of the Offer Object:

[https://schema.org/hasMerchantReturnPolicy](https://schema.org/hasMerchantReturnPolicy)
[https://schema.org/shippingDetails](https://schema.org/shippingDetails)

I'm referring in concrete to these two properties because [Google Search Console](https://developers.google.com/search/docs/appearance/structured-data/product?hl=en#merchant-listing-experiences) is showing the lack of them in a Product -> Offers as an issue (not critical) and it would be nice to have them

Many thanks!

### Schema objects

https://schema.org/Offer

**Comment by Turnerj** [comment]
Yeah, we're a bit behind with schema.org version updates. As far as I can tell, we're currently on version 12 with version 16 being released yesterday (which is great timing then for your issue).

I've got #590 open to update the schema.org version. One thing I will note though, both of those properties are new/pending on schema.org so they will only be in [Schema.NET.Pending](https://www.nuget.org/packages/Schema.NET.Pending) rather than the main library once my PR is merged and a new release is done.

**Comment by Turnerj** [comment]
I'm going to close this now that #590 is merged however there isn't a new release of Schema.NET just yet.
